src/nuclide.cpp

```diff
@@ -678,9 +699,21 @@ void Nuclide::calculate_xs(int i_sab, int i_log_union, double sab_frac, Particle
       // Calculate microscopic nuclide nu-fission cross section
       micro.nu_fission = (1.0 - f)*xs(i_grid, XS_NU_FISSION)
         + f*xs(i_grid + 1, XS_NU_FISSION);
+      micro.prompt_nu_fission = (1.0 - f)*xs(i_grid, XS_PROMPT_NU_FISSION)
+        + f*xs(i_grid + 1, XS_PROMPT_NU_FISSION);
+      micro.delayed_nu_fission.resize(n_precursor_);
```

Comment:
It's possibly inefficient to call vector's `resize` in performance-critical code like this. I don't think in practice you'd see this because `vector`s tend to not let go of memory they've already allocated, at least with gcc's STL (although this is not guaranteed and may vary between C++ STL implementations AFAIK).

If you do end up sticking with `std::vector` in the micro XS cache, I recommend calling `resize` up front on each particle, in the constructor for this rather than here.